vrc-get-gui/locales/zh_cn.json5

```diff
@@ -53,16 +53,16 @@
     "projects:pre-migrate copying...": "复制项目以进行迁移...",
     "projects:migrating...": "迁移项目中...",
     "projects:type:legacy": "旧版",
-    "projects:button:open unity": "打开 Unity",
+    "projects:button:open unity": "打开项目",
```

Comment:
I don't quite agree with you, because the translated meaning exactly according to the original string tends to have comprehension problems in Chinese

Reply:
I machine translated "打开项目" and it says "Open it" I say based on this.

I think Open it lacks how open the project. We can open the project folder with explorer/Finder to edit non project files, open in  ALCOM to manage packages, and open unity to build something.

Reply:
> I machine translated "打开项目" and it says "Open it" I say based on this.
> 
> I think Open it lacks how open the project. We can open the project folder with explorer/Finder to edit non project files, open in ALCOM to manage packages, and open unity to build something.

@anatawa12 What kind of translation are you using? I'm try to use Google Translate.In Google Translate “打开项目” it means "Open Project"
![photos](https://github.com/vrc-get/vrc-get/assets/47104993/24108481-b7b1-4955-bf73-26b42cefbd06)

Reply:
就像 @anatawa12 所说
> I think Open it lacks how open the project. We can open the project folder with explorer/Finder to edit non project files, open in ALCOM to manage packages, and open unity to build something.
如果为“打开项目”则有可能有多种打开方式
所以我建议修改为“使用 Unity 打开”。 @flower-elf

Reply:
agree

Reply:
但这样可能会显得按钮会很长，为了尽可能缩短按钮，维持原文也不失为一个很好的选择
However, this may appear that the buttons will be very long, and it is not a bad idea to maintain the original text in order to shorten the buttons as much as possible

Reply:
> 但这样可能会显得按钮会很长，为了尽可能缩短按钮，维持原文也不失为一个很好的选择 However, this may make that the buttons will be very long, and it is not a bad idea to maintain the original text in order to shorten the buttons as much as possible

因为原文有不少人说看着觉得奇怪，所以才打算修改，如果这确实破坏了布局的话，就先暂时维持原文吧。

Reply:
Sorry I used CN -> JP google translation and it says "項目を開く" and it means Open item or open it.

Reply:
> Sorry I used CN -> JP google translation and it says "項目を開く" and it means Open item or open it.

Thank you very much for your reply.(。・∀・)ノ
Due to display issues I will take @lonelyicer advice and keep the original translation "打开 Unity" to avoid some problems.